common/test/acceptance/pages/studio/overview.py

```diff
@@ -575,6 +575,14 @@ def change_problem_due_date(self, date):
         self.q(css=".action-save").first.click()
         self.wait_for_ajax()
 
+    def select_visibility_tab(self, desired_item='hide_after_due_date'):
+        """
+        Select the advanced settings tab
+        """
+        self.q(css=".settings-tab-button[data-tab='visibility']").first.click()
+        if desired_item == 'hide_after_due_date':
```

Comment:
You can remove this check for desired_item (and the option of passing it in). In all places where `test_answer_says_submitted` is called, you are expecting the default value of `'hide_after_due_date'`.